### DISCUSSION TITLE
Trying to jump multiple items at a time.

### DISCUSSION THREAD

**Discussion by Archie-2021** [gokcehan/lf#1363] [Q&A] [answered]
I am trying to use this , but everytime I hit `J` or `K` it jumps into terminal and comes back, is there a way to stop that?

The code:

```
map J go6down
map K go6up

cmd go6down ${{
  lf -remote "send $id down" 
  lf -remote "send $id down" 
  lf -remote "send $id down"
  lf -remote "send $id down"
  lf -remote "send $id down"
  lf -remote "send $id down"
}}

cmd go6up ${{
  lf -remote "send $id up" 
  lf -remote "send $id up" 
  lf -remote "send $id up" 
  lf -remote "send $id up" 
  lf -remote "send $id up" 
  lf -remote "send $id up" 
}}
```

**Comment by joelim-work** [comment, marked as answer]
`$` commands are synchronous, which means it suspends the UI and shows the shell command execution until it finishes. If you don't want that then you can use `%` or `&`, which runs the shell command asynchronously in background (`%` reads from and writes to the bottom statline).

That being said, you probably don't need to use `lf -remote` here - you can just use `push` to simulate key presses, which is probably a lot cleaner.

```
map J push 6j
map K push 6k
```

**Comment by Archie-2021** [comment]
Thanks you very much. Really enjoying to learn about `lf`.  One of the best programs ever.
